docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -105,24 +157,33 @@ Instead, set up limited demo accounts, or fake credit cards with limited functio
 ====
 
 Use environment variables to handle any values needing encryption at rest. 
-The example below uses the syntax `${URL}` to reference a variable named `URL` defined in the environment:
+The example below references a variable named `MY_URL` defined in the environment:
```

Comment:
it's not immediately clear to me that this example is demonstrating how to do this for project monitors and the file is intended to be a `synthetics.config.ts` file. Can we make that more clear?